model/riscv_insts_zfa.sail

```diff
@@ -52,7 +52,7 @@ function clause execute (RISCV_FLI_H(constantidx, rd)) = {
     0b11100 => { 0x7800 },  /* 2^15 */
     0b11101 => { 0x7c00 },  /* 2^16 */
     0b11110 => { 0x7c00 },  /* +inf */
-    0b11111 => { canonical_NaN_H() },
+    0b11111 => { canonical_NaN(16) },
```

Comment:
Why have we lost the suffixed versions? It’s nicer being able to consistently name things with _H etc rather than passing in bit widths as numbers.

Reply:
(And reduces churn if you keep them too)

Reply:
Ok I kept both versions and defined the generic one in terms of the suffixed ones.